SECURITY.md

```diff
@@ -0,0 +1,83 @@
+# Security Policy
+
+## Supported Versions
+
+Currently supported versions for security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.7.x   | :white_check_mark: |
+| < 0.7   | :x:                |
+
+## Reporting a Vulnerability
+
+We take security seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue for the vulnerability
+2. Email security details to the maintainers (see repository contacts)
+3. Include:
+   - Description of the vulnerability
+   - Steps to reproduce
+   - Potential impact
+   - Suggested fix (if available)
+
+### Response Timeline
+
+- **Initial Response**: Within 48 hours
+- **Status Update**: Within 7 days
+- **Resolution Target**: Within 30 days for critical issues
+
+## Security Considerations
```

Comment:
`SECURITY.md` is where you look after you've found a vulnerability. <https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository>

Move everything unrelated to reporting a vulnerability to library documentation.